src/components/App/App.js

```diff
@@ -0,0 +1,210 @@
+            {sortedSymptoms.map((item, index) => {
+              return (
+                <div className={"answer " + (this.state.chosenSymptomId != null ? "disabled" : "")}
+                      onClick={this.toggleSymptom}
+                      id={index}
+                      key={item.id}>
+                  {item.name}
+                </div>
+              )
+            })}
+          </div>
+        </div>
+
+        {this.state.chosenSymptomId != null &&
+          <div>
+            <QuestionContainer text={"Do you have " + sortedSymptoms[this.state.chosenSymptomId].diagnoses[0].name + "?"} />
+            <div className="question-container">
+              <div className="individual-question-container answer-container">
+                <PersonIcon diasble={this.state.rightDisease != null }/>
+                <div className={"answer " + (this.state.rightDisease != null ? "disabled" : "")}
+                     onClick={this.isRightDisease}
+                     id="false"
+                     disease={null}>
+                  No</div>
+                <div className={"answer " + (this.state.rightDisease != null ? "disabled" : "")}
+                     onClick={this.isWrongDisease}
+                     id={sortedSymptoms[this.state.chosenSymptomId].diagnoses[0].id}>
+                  Yes</div>
+              </div>
+            </div>
+          </div>
+        }
+
+        {this.state.rightDisease === 'false' &&
+          <div>
+            <QuestionContainer text="That's the most common disease with that symptom, 
+                  but these diseases are also very common. Please click on the disease that you do have." />
+            <div className="question-container">
+              <div className="individual-question-container answer-container">
+                <PersonIcon diasble={this.state.correctDisease != null }/>
+                {sortedSymptoms[this.state.chosenSymptomId].diagnoses.map((item, index) => {
+                  return (
+                    <div className={"answer " + (this.state.correctDisease != null ? "disabled" : "")}
+                         onClick={this.chooseDisease}
+                         id={item.id}
+                         key={item.id}>
+                      {item.name}
+                    </div>
+                  )
+                })}
+              </div>
+            </div>
+          </div>
+        }
+
+        {this.state.correctDisease != null &&
+          <div>
+            <QuestionContainer text="Great, glad we figured out what you have.  That much be a relief." />
+            <QuestionContainer text={"There " + (diagnoses[0].frequency === 1 ? "has" : "have") + " been " + 
+                                  diagnoses[0].frequency + " case" + (diagnoses[0].frequency === 1 ? "" : "s") + 
+                                  " of " + diagnoses[0].name + " since this server was reset."}/>
+            <div className="question-container">
```

Comment:
I see a QuestionContainer component above. From the looks of it you could have passed the contents of this div to that component and allowed it to take optional children (it's confusing re-using a class that's also the name of a component)

Reply:
This wouldn't work actually- the QuestionContainer has different class names than the answers.  I used the class name in both because they share some but not all styling.  

I definitely agree that the App file could deal with some cleanup.  If I had more time, I would have liked to break out the components a bit more.

Reply:
The only additional class I'm seeing is answer-container? Could be added w/ a prop.

Reply:
Not that this is a huge deal to be honest.